### ISSUE TITLE
NVDA Does Not Work With Tables In Chrome

### ISSUE THREAD

**Issue by MRayermannMSFT** [nvaccess/nvda#8160]
### Steps to reproduce:
1. Download and extract test.zip
2. Open test.html in Google Chrome
3. Navigate to the table inside the document
4. Try to use control + alt +arrow keys to navigate the table

### Expected behavior:
I can navigate the table

### Actual behavior:
I cannot navigate the table.

### System configuration:
NVDA version: 2018.1.1

NVDA Installed or portable: installed

Other information:
- The issue does not reproduce on Firefox.
- On Firefox, when I navigate backwards through the document, NVDA reads the entire content of the table. This is probably an entirely different issue though.
- Feel free to let me know if my HTML is somehow wrong.

Windows version: Windows 10 1709

Name and version of other software in use when reproducing the issue: Google Chrome

### Other questions:
N/A

Does the issue still occur after restarting your PC?
Yes

Have you tried any other versions of NVDA?
NVDA 2017 (minor number and patch number unknown)

[test.zip](https://github.com/nvaccess/nvda/files/1895538/test.zip)

**Comment by ehollig** [comment]
@MRayermannMSFT, I am unable to replicate your issue. When I view the attached document in Chrome, I am able to navigate the table using table commands. One thing I did notice was, NVDA does not announce the "aria-controls" associated with the table headers, but that is probably a separate issue.
